src/interaction/EfInteraction.py

```
@@ -143,6 +162,9 @@ def send_key_down(self, key, activate=True):
                 self.make_lparam(vk_code, is_up=False),
             )
             return
+        # 后台模式下：按下时前置游戏（pynput 只投递到前台窗口），松开时恢复原窗口
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)